### PR TITLE
feat (localization): Add FE translations for italian

### PR DESCRIPTION
**Pull request by lovrocolic**
## Context

Add possibility to set italian as default locale for documemts. Also, add italian translations for email preview section

### CODE REVIEW

**Review by ansmonjol — APPROVED** [review]

File: src/components/customerPortal/PortalInvoiceListItem.tsx [hunk 1]
```diff
@@ -49,6 +49,7 @@ interface PortalInvoiceListItemProps {
   invoice: PortalInvoiceListItemFragment
   translate: Function
   className?: string
+  documentLocale: string
```
Comment (ansmonjol) [inline src/components/customerPortal/PortalInvoiceListItem.tsx:52]: Should be of type LocaleEnum

File: src/pages/auth/PortalInit.tsx [hunk 2]
```diff
@@ -43,11 +43,11 @@ const PortalInit = () => {
     nextFetchPolicy: 'network-only',
     skip: !isPortalAuthenticated || !token,
   })
-  const { translateWithContextualLocal: translate } = useContextualLocale(
-    (data?.customerPortalUser?.billingConfiguration?.documentLocale as Locale) ||
-      (data?.customerPortalOrganization?.billingConfiguration?.documentLocale as Locale) ||
-      'en'
-  )
+
+  let documentLocale = (data?.customerPortalUser?.billingConfiguration?.documentLocale as Locale) ||
```
Comment (ansmonjol) [inline src/pages/auth/PortalInit.tsx]: You can set it as a const. It won't be reassigned

File: src/components/customerPortal/PortalInvoiceListItem.tsx (hunk 1 above)
Comment (ansmonjol) [reply src/components/customerPortal/PortalInvoiceListItem.tsx:52]: if it complain about the 'en' being a non valid value, we need to update the LocaleEnum definition to be an object `as const`

Let me know if that's happening

Comment (lovrocolic) [reply src/components/customerPortal/PortalInvoiceListItem.tsx]: changed

**Pull request merged**